Resources/doc/index.rst

```diff
@@ -50,7 +50,7 @@ Require the bundle with composer:
 
 .. code-block:: bash
 
-    $ composer require friendsofsymfony/user-bundle "~2.0@dev"
+    $ composer require friendsofsymfony/user-bundle "dev-master"
```

Comment:
I wouldn't do that. This won't restrict your versions to 2.x anymore, but could pull in 3.0 dev versions (once they are started) which can lead to totally unexpected behaviour.